scripts/spack/packages/sundials/package.py

```diff
@@ -16,134 +18,167 @@ class Sundials(CMakePackage, CudaPackage, ROCmPackage):
     homepage = "https://computing.llnl.gov/projects/sundials"
     url = "https://github.com/LLNL/sundials/releases/download/v2.7.0/sundials-2.7.0.tar.gz"
     git = "https://github.com/llnl/sundials.git"
-    tags = ['radiuss', 'e4s']
+    tags = ["radiuss", "e4s"]
+    test_requires_compiler = True
 
-    maintainers = ['balos1', 'cswoodward', 'gardner48']
+    maintainers("balos1", "cswoodward", "gardner48")
 
     # ==========================================================================
     # Versions
     # ==========================================================================
-    version('develop', branch='develop')
+    version("develop", branch="develop")
```

Comment:
The only project specific thing in this file is the asan variant. I updated it to spack proper and added the new version and fixed a new bug in it. I pushed that up to Spack and its in a PR.